main/default.cfg

```diff
@@ -7,65 +7,46 @@ unbindall
 //
 // WEAPONS/UPGRADES
 //
+
 bind  MOUSE1        +attack
 bind  MOUSE2        +attack2
 
-bind  [             weapprev
-bind  ]             weapnext
 bind  mwheelup      weapprev
 bind  mwheeldown    weapnext
-bind  ENTER         +useitem
 bind  MOUSE3        +useitem
 bind  g             itemact grenade
+bind  f             itemact medkit
+bind  1             itemtoggle blaster
+bind  r             reload
 
 //
 // CHARACTER CONTROLS
 //
 
-bind  CTRL          +attack
-
-bind  ALT           +strafe
-bind  SHIFT         +speed
-bind  x             +sprint
-
-bind  PGDN          +lookdown
-bind  PGUP          +lookup
-bind  END           centerview
+bind  ALT         	+speed
+bind  SHIFT         +sprint
 
-bind  c             +movedown
+bind  CTRL          +movedown
 bind  SPACE         +moveup
 
-bind  UPARROW       +forward
-bind  DOWNARROW     +back
-bind  LEFTARROW     +left
-bind  RIGHTARROW    +right
+
 bind  w             +forward
 bind  a             +moveleft
 bind  s             +back
 bind  d             +moveright
 
-bind  ,             +moveleft
-bind  .             +moveright
 
 //
 // MISC
 //
 
-bind  t             message_public
+bind  ENTER         message_public
```

Comment:
Let's keep these keys next to each other. If we use enter to initiate chat, we should have a way to change visibility of the chat from inside the chat prompt.